buildenv/jenkins/triggerGrinder.groovy

```diff
@@ -0,0 +1,69 @@
+#!groovy
+
+String json_path = "../2072_test.json"
+File json = new File(json_path)
+
+def jenkins_url = "https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/buildWithParameters"
+def get_key_value_regex = ~/"(?<key>[^"]*)"\s*:\s*"(?<value>[^"]*)"/
+def key, value
+def should_run_grinder = false // Assume that the job should not be run again
+def map = [:]
+
+/**
+ This runs when we have found a job w/ git_issue_status = closed.
+ Collects all parameters and runs grinder 
+ */
+def run_grinder(map) {
+    def jenkins_url = "https://ci.adoptopenjdk.net/view/" + 
+                      "Test_grinder/job/Grinder/buildWithParameters"
+    def list = []
+    def queryString
+
+    map.each { k, v -> 
+        list.add(k + "=" + v)
+    }
+    
+    queryString = list.join("&")
+    url = jenkins_url + "/?" + queryString
+
+    // The Grinder trigger will look something like this, 
+    // but it requires tokens which I do not have yet.
+    // def url = new URL(jenkins_url + "/?" + queryString)
+    // def connection = url.openConnection()
+    // connection.with {
+    //     doOutput = true
+    //     requestMethod = 'POST'
+    //     outputStream.withWriter { writer ->
+    //         writer << queryString
+    //     }
+    //     println content.text
+    // }
+
+    println url
```

Comment:
Ideally, we should not use a remote trigger. We can use jenkins API, so no credentials are needed. I am ok with the print URLs for now and we can enhance this later.